GibbonEdu/core
# Updated JQuery and JQuery Migrate Files

**Description**
Updated: 
JQuery: v2.2.4 to v3.7.1  
JQuery Migrate: v1.4.1 to v3.4.0

The new JQuery versions were copied into their corresponding JQuery files located at: 
`gibbon/lib/jquery/jquery-migrate.min.js`
`gibbon/lib/jquery/jquery.js`

**Motivation and Context**
This update was prompted by certain vulnerabilities being pointed out to us, which was a result of an outdated version of JQuery and JQuery Migrate.

**How Has This Been Tested?**
Tested locally with these features in mind:
- Check Boxes
- Bulk Actions
- Highlighting Pre-Checked Rows
- Form Class: Generic Check all/none checkboxes
- Column Highlighting
- Password Generator
- Username Generator
- Data Table: Expandable Rows
- Forms: Expandable Rows

NOTE: These are features mentioned in the `core.js` file that were not able to be tested due to an unrelated bug in my development environment:
- Toggle Switch
- Sidebar Toggle Switch
- Data Table: Simple Drag and Drop
